tailcfg/tailcfg.go

```diff
@@ -1060,6 +1060,13 @@ type Debug struct {
 	// :0 to get a random local port, ignoring any configured
 	// fixed port.
 	RandomizeClientPort bool `json:",omitempty"`
+
+	/// UPnPDisabled is whether the client will attempt to perform a UPnP portmapping.
```

Comment:
this is backwards.

And maybe as well make it match the naming and doc wording of `DisableSubnetsIfPAC` above.

Oh, the var IS named `Disable` first. But the doc subject doesn't match. Just fix the docs.